### PR TITLE
Comparing case sensitive issuing addresses

### LINKED ISSUE #25: Allow comparing of case sensitive Ethereum addresses
Not sure if cert-verifier-js has ever ran into a bitcoin address compare issue at all, but for ethereum, if the issuing address is 0xDeaDDeaD... but a blockchain explorer api returns 0xdeaddead... as the address that issued the certificate, then the `ensureValidIssuingKey()` method will fail. 

In Ethereum, case sensitive addresses only matter for checksum checking opportunities, but otherwise resolve to the same all lower case equivalent version of the address.

### PR DESCRIPTION
**Pull request by AnthonyRonning**
Fixes #25

### CODE REVIEW

File: lib/checks.js [hunk 1]
```diff
@@ -246,4 +247,11 @@ function _hexFromByteArray(byteArray) {
     }
   }
   return out;
-};
\ No newline at end of file
+};
+
+function getCaseInsensitiveKey(obj, name) {
+  var realName = _.findKey(obj, function (value, key) {
```
Comment (raiseandfall) [inline lib/checks.js]: @AnthonyRonning We should probably do that in pure JS. Adding the `underscore` library to the bundle just for that seems like a bit much.

Comment (raiseandfall) [reply lib/checks.js]: https://gist.github.com/DaveAtDog/11207429

Comment (AnthonyRonning) [reply lib/checks.js]: Thanks for that link, altered it a bit so it could be used to find case insensitive keys as well. Removed underscore.

Comment (raiseandfall) [reply lib/checks.js]: @AnthonyRonning thanks for adding it!

**Pull request merged**